index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- <div class="d-flex flex-center"> -->
+        <header class="header d-flex flex-center">
+            <a href="#">Alejandro Ramos</a>
+            <a class="burger" href="#"><img src="images/IconMenu.png" alt="burger button"></a>   
+        </header>
+    <!-- </div> -->
+    <section class="mainContain d-flex">
+        <div class="informativeText">
+            <h1>Hey There.<br>I'm Alex</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product, feature or<br>website Look through some of my work<br>and experience! If you like what you see<br>and have a project you need coded,<br> 
+            don’t hestiate to contact me.</p>
+        </div>
+        <div class="socialIcons">
+            <ul class="d-flex">
+                <li>
+                    <a href="https://github.com/AlexRS90">
+                        <img src="images/IconGitHub.png" alt="-GitHub button">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/alejandro-ramos-santos-9b0b52135/">
+                        <img src="images/IconLinkedin.png" alt="Linkedin button">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="images/IconAngelist.png" alt="Angelist button">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/AlejandroRBenji">
+                        <img src="images/IconTwitter.png" alt="Twitter button">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="images/IconMedium.png" alt="Medium button">
+                    </a>
+                </li>
+            </ul>
+        </div>        
+    </section>
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to wrap the entire `section` in a root `main` tag as pointed out by the previous reviewer

Reply:
Hello Ted! 
Thanks for your suggestion, I have a doubt, the root main tag should be inside of the body or even the body must be inside the main tag? 

Thanks in advance!!

Reply:
As you probably know, the fundamental **HTML document format** is like so:

```
<!DOCTYPE html>
<html>
  <head>
  </head>
  <body>
  </body>
</html>
```

So, the `main` tag and other semantic tags should **ALWAYS** be in the `body` tag.
Just wrap the `section` in the `main` tag like so:

```
...
<main>
  <section>
    ...
  </section>
</main>
...

Reply:
Got it!

Really really thanks! 🤝